src/controllers/midi/midicontroller.cpp

```diff
@@ -193,6 +198,15 @@ QString formatMidiMessage(unsigned char status, unsigned char control, unsigned
                          QString::number((status & 255)>>4, 16).toUpper(),
                          QString::number(control, 16).toUpper().rightJustified(2,'0'),
                          QString::number(value, 16).toUpper().rightJustified(2,'0'));
+        case MIDI_START:
+            return QString("MIDI status 0x%1: Start Sequence")
+                    .arg(QString::number(status, 16).toUpper());
+        case MIDI_TIMING_CLK:
```

Comment:
This is the name of the message in the midi spec: https://www.cs.cf.ac.uk/Dave/Multimedia/node158.html "Timing Clock: F8".  Note that midi has more than one timing mechanism, so it's important to stay close to the spec instead of inventing our own terminology. https://www.google.com/search?client=ubuntu&channel=fs&q=midi+timing+clock+f8&ie=utf-8&oe=utf-8